geoapps/drivers/components/data.py

```diff
@@ -16,15 +15,43 @@
 
 from geoapps.utils import calculate_2D_trend, filter_xy
 
+from .factories import SimulationFactory, SurveyFactory
 from .locations import InversionLocations
+from .meshes import InversionMesh
 
 
 class InversionData(InversionLocations):
-    """ Retrieve data from workspace and apply transformations. """
+    """
+    Retrieve data from workspace and apply transformations.
+
+    Methods
+    -------
+
+
+    """
+
+    def __init__(self, workspace, params, window):
+        """
+        :param: resolution: Desired data grid spacing.
```

Comment:
workspace and window missing from docstring. For all the params, I am not sure how this should be written. Technically we only need to docs the Params input... but I get that it needs specific fields...

Reply:
Maybe I should leave out any parameters not in the call signature, but create a list of parameters in the class docstring.

Reply:
Overhauled the docstrings so that class level docstrings handle the attributes not assigned during instantiation and not the init docstring.  Also added a methods list to class level docstrings.  I used the Numpy/Scipy docstring format for the class level docstrings since there seems to be no documented way to handle methods in reformated text.

Reply:
Ok for now. See GEOPY-279